Updates to the PostHog vs Amplitude blog post (#12884)
PostHog/posthog.com: contents/blog/posthog-vs-amplitude.mdx

Reviewer: @natalia-amorim we should get your byline on this. we may need to set you up as an author, though

**PostHog**. [Usage-based pricing](/pricing), ability to cap spend, and transparent per-unit rates.    

</details>
<details>
  <summary>Best for scaling B2C apps with very high MTUs</summary>

**Amplitude**. MTU-based pricing can be more predictable and cost-effective if you track millions of users with relatively few events.  

</details>
<details>
  <summary>Best for engineers and technical founders</summary>

**PostHog**. Open source, transparent roadmap, direct support from engineers, SQL access, and extensibility.  

</details>
<details>
  <summary>Best for warehouse-native analytics</summary>

**Amplitude**. Can run directly on top of your data warehouse, while PostHog syncs to/from warehouses but doesn't sit on top of them.  

</details>
<details>
  <summary>Best for account-based analytics (B2B SaaS)</summary>

**PostHog**. Group/account analytics are included in the free plan, and [Revenue analytics](/docs/revenue-analytics/) *(beta)* lets SaaS teams connect tools like Stripe, track recurring revenue, and analyze top customers. Amplitude requires a paid add-on for account analytics.  

</details>
<details>
  <summary>Best all-in-one platform</summary>

> **PostHog**. PostHog** – combines web and product analytics, feature flags, replays, surveys, error tracking, and a data warehouse in one product.

Reviewer 2: one more typo

> **PostHog**. PostHog** – combines web and product analytics, feature flags, replays, surveys, error tracking, and a data warehouse in one product.

Reviewer 2: ont more typo

```suggestion
**PostHog**. Combines web and product analytics, feature flags, replays, surveys, error tracking, and a data warehouse in one product.
```

import { ComparisonTable } from 'components/ComparisonTable'
import { ComparisonHeader } from 'components/ComparisonTable/header'
import { ComparisonRow } from 'components/ComparisonTable/row'
import { CalloutBox } from 'components/Docs/CalloutBox'

> import { AskMax } from 'components/AskMax'

Reviewer 2: Remove the curly-s

```suggestion
import AskMax from 'components/AskMax'
```

</details>
<details>
  <summary>Best for warehouse-native analytics</summary>

**Amplitude**. Can run directly on top of your data warehouse, while PostHog syncs to/from warehouses but doesn't sit on top of them.  

</details>
<details>
  <summary>Best for account-based analytics (B2B SaaS)</summary>

**PostHog**. Group/account analytics are included in the free plan, and [Revenue analytics](/docs/revenue-analytics/) *(beta)* lets SaaS teams connect tools like Stripe, track recurring revenue, and analyze top customers. Amplitude requires a paid add-on for account analytics.  

</details>
<details>
  <summary>Best all-in-one platform</summary>

**PostHog**. Combines web and product analytics, feature flags, replays, surveys, error tracking, and a data warehouse in one product.

</details>

Whichever you choose, setup is quick. Amplitude offers a polished SaaS experience with strong documentation and enterprise support. PostHog makes setup fast with a snippet or wizard install, automatic event capture, and templates (plus [AI assistance with MaxAI](/max)) to get insights running quickly.

Have a direct question? Just ask Max.

<AskMax

>     title="PostHog or Amplitude?"

Reviewer 2: remove title, it's not supported in the `<AskMax>` component rn

```suggestion
```

    />
    <ComparisonRow
        column1={true}
        column2={true}
        feature="User property targeting"
        description="Target users based on any of their user properties"
    />
    <ComparisonRow
        column1={true}
        column2={true}
        feature="Custom colors & positioning"
        description="Customize the colors of your surveys to match your brand"
    />
    <ComparisonRow
        column1={true}
        column2={true}
        feature="Custom HTML"
        description="Add custom HTML to your survey text"
    />
    <ComparisonRow column1={true} column2={false} feature="API mode" description="Create surveys via the API" />
</ComparisonTable>

<CalloutBox icon="IconInfo" title="Good to know" type="fyi">

[Survey templates](/templates?filter=type&value=survey) make it easy to run NPS, product-market-fit (PMF), and customer satisfaction (CSAT) surveys in just a few clicks. Read our [guide comparing NPS, CSAT and CES](/product-engineers/nps-vs-csat-vs-ces) to for more on how to use surveys.

</CalloutBox>

## Price comparison

### PostHog pricing philosophy

> **PostHog charges based on usage**. Each product (analytics, session replay, feature flags, etc.) comes with a [generous free tier](/pricing), and once you exceed those limits you pay only for what you use. Pricing is fully transparent and published publicly, with per-unit rates you can calculate in advance. You can also set billing caps per product to avoid unexpected costs.

Reviewer: ```suggestion
**PostHog charges based on usage**. Each product (analytics, session replay, feature flags, etc.) comes with a [generous free tier](/pricing), and once you exceed those limits you pay only for what you use. 

Pricing is fully transparent and published publicly, with per-unit rates you can calculate in advance. You can also set billing caps per product to avoid unexpected costs.
```

> This usage-based approach means costs scale with your actual activity, whether that's events, surveys, or recordings, rather than the number of users in your product.

Reviewer: ```suggestion
This usage-based approach means costs scale with your actual activity, whether that's events, surveys, or recordings, rather than the number of users in your product. It also means you can easily reduce your bill by changing how you track users to send fewer events, or record fewer sessions.
```

### Amplitude pricing philosophy

**Amplitude charges based on Monthly Tracked Users (MTUs)**. Each tracked user is counted once per month, regardless of how many events they generate. Plans scale as your MTUs grow, with higher tiers unlocking more advanced analytics, experimentation, and enterprise features.

> This model can be easier to predict if you run a large product with millions of users who generate relatively few events per person. It also means teams don't have to think about event volume when forecasting costs. However, it can feel less flexible if you're working with a smaller but more active user base, since a few highly engaged users won't change your MTU count.

Reviewer: ```suggestion
This model can be easier to predict if you run a large product with millions of users, but it's less flexible because there's no way to reduce how much you spend. Every user is counted equally regardless of how valuable they are to you.
```

@ivanagas interested in your take here – I think this is a better way describe this difference.

<br />

## TL;DR: Which one is better, PostHog or Amplitude?

Both tools cover the core product analytics stack, but the right choice varies depending on what you're optimizing for:

**Best product analytics tool for startups: PostHog** – generous free tiers, $50k startup credits for eligible companies, and usage-based pricing keep costs low as you grow.

**Best for teams with limited engineering resources: Amplitude** – polished SaaS setup with strong documentation and established enterprise support.

**Best for engineers and technical founders: PostHog** – open source, transparent roadmap, direct support from engineers, SQL access, and extensibility.

**Best for account-based analytics (B2B SaaS): PostHog** – group/account analytics included in the free plan, while Amplitude requires a paid add-on.

**Best for classic marketing analytics:** **Amplitude** – mature dashboards and integrations designed for large product/marketing teams.

**Best for flexibility and cost control**: **PostHog** – usage-based pricing, ability to cap spend, and transparent per-unit rates.

> **Best all-in-one platform** – **PostHog**: combines analytics, feature flags, replays, surveys, error tracking, and a data warehouse in one product.

Author: ```suggestion
**Best all-in-one platform: **PostHog** – combines analytics, feature flags, replays, surveys, error tracking, and a data warehouse in one product.
```
